libs/sdk/e2e-schematics/src/generators/component-e2e/index.ts

```diff
@@ -254,14 +254,20 @@ export default async function (tree: Tree, schema: Partial<Schema>) {
   const storybookVersion =
     packageJson.devDependencies['@storybook/addon-toolbars'];
   if (storybookVersion) {
+    // The `addDependenciesToPackageJson` will not set the right version if the version is lower than the version than what the `storiesGenerator`
+    // installs (even if this version uses a caret). Remove the entries first so that the addition will go through with the right version.
+    removeDependenciesFromPackageJson(
```

Comment:
The note calls this out - but NX changed how the `addDependenciesToPackageJson` to now no install things if the given version is lower than the currently installed version. They did this without caring about carets and so in order to maintain our old behavior - we have to first remove the installed version (via the component generator) and then add it back with the version we want.